portfolio/src/main/java/com/google/sps/servlets/RandomMessageServlet.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+import com.google.sps.data.Message;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Random;
+
+/** Servlet responsible for listing tasks. */
+@WebServlet("/random-message")
+public class RandomMessageServlet extends HttpServlet {
+
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    Random rand = new Random();
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html;");
+        Query<Entity> query = Query.newEntityQueryBuilder().setKind("Message").build();
+        QueryResults<Entity> results = datastore.run(query);
+
+        List<String> messages = new ArrayList<>();
+        while (results.hasNext()) {
+            Entity entity = results.next();
+
+            String messageContent = entity.getString("messageContent");
+
+            messages.add(messageContent);
+        }
+        String messageToDisplay = messages.get( rand.nextInt(messages.size()));
```

Comment:
What happens if your datastore doesn't have any messages stored in it?

Reply:
I think that it would crash :0. I'm going to fix it! Thanks for reviewing my code!